pytest_instafail.py

```diff
@@ -21,13 +21,42 @@ def pytest_addoption(parser):
             "default)."
         )
     )
+    try:
+        parser.addini(
+            "instafail", default=False, type="bool",
+            help=(
+                "show failures and errors instantly as they occur (disabled by "
+                "default)."
+            )
+        )
+    except AssertionError:
+        # This is pytest < 2.9.0
+        parser.addini(
+            "instafail", default=False, type=None,
+            help=(
+                "show failures and errors instantly as they occur (disabled by "
+                "default)."
+            )
+        )
```

Comment:
I think it would be better to have the (kw)args only in one place, and then retry with `type=None` replaced in that.